src/library/Uno.Themes.WinUI.Markup/Theme.LightWeight/Themes.LightWeightStyles.Typography.cs

```diff
@@ -0,0 +1,220 @@
+﻿using System;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup.Internals;
+using Windows.UI.Text;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static class Typography
+		{
+			public static class Resources
+			{
+				public static class DisplayLarge
+				{
+					[ResourceKeyDefinition(typeof(string), "DisplayLargeFontWeight")]
+					public static ResourceValue<string> FontWeight => new("DisplayLargeFontWeight", true);
```

Comment:
```suggestion
					[ResourceKeyDefinition(typeof(FontWeight), "DisplayLargeFontWeight")]
					public static ResourceValue<FontWeight> FontWeight => new("DisplayLargeFontWeight", true);
```

Same for the other FontWeight @Arieldelossantos

Reply:
fixed... no needed the #if WinUI/Uwp conditional for the FontWeight, because we'll only support WinUI